raiden/connection_manager.py

```diff
@@ -233,11 +233,11 @@ def join_channel(self, partner_address: Address, partner_deposit: TokenAmount) -
         # Wait for any pending operation in the channel to complete, before
         # deciding on the deposit
         with self.lock, token_network_proxy.channel_operations_lock[partner_address]:
-            channel_state = views.get_channelstate_for(  # type: ignore # FIXME: arg 2 looks wrong!
-                views.state_from_raiden(self.raiden),
-                self.token_network_address,
```

Comment:
This tells me that we lack tests for the connection manager that this was hidden this whole time. Could you please create an issue for adding more tests for the connection manager?